composer.json

```diff
@@ -30,12 +30,10 @@
         "phpunit/PHPUnit": "~4.0"
     },
     "suggest": {
-        "ext-mongo": "*",
-        "zendframework/zend-console": "Zend\\Console component",
-        "zendframework/zend-db": "Zend\\Db component",
```

Comment:
> but in this component exist a mysql writer?

No MySQL, but a [database writer](http://framework.zend.com/manual/current/en/modules/zend.log.writers.html#writing-to-databases).